src/test/java/pageobject_model/test/Helpers.java

```diff
@@ -37,4 +41,12 @@ public static void fillTextField(WebDriver driver, String locator, WebElement te
         textField.click();
         textField.sendKeys(text);
     }
+
+    public static void takeScreenshot() throws AWTException, IOException {
+        Robot robot = new Robot();
+        java.awt.Rectangle screenSize = new Rectangle(Toolkit.getDefaultToolkit().getScreenSize());
+        BufferedImage image = robot.createScreenCapture(screenSize);
+        ImageIO.write(image, "png", new File("C:\\Users\\Andrii_Prokofiev1\\IdeaProjects\\WebdriverGridOrigin\\src\\test\\java\\pageobject_model\\test\\Screenshot\\"));
```

Comment:
It's always better to use not static but relative location, there are even some recommendation to test output in jave.